kotlin/samples/hello-compose-binding/src/androidTest/java/com/squareup/sample/hellocomposebinding/HelloBindingTest.kt

```diff
@@ -28,7 +28,7 @@ import org.junit.runner.RunWith
 class HelloBindingTest {
 
   // Launches the activity.
-  @Rule @JvmField val scenario = ActivityScenarioRule(HelloBindingActivity::class.java)
+  @Rule @JvmField val composeRule = AndroidComposeTestRule<HelloBindingActivity>()
```

Comment:
Compose top-level UI test functions now require a `ComposeTestRule` to be used, or they will throw.